lib/fathead/perl_doc/Parsers/parse.pl

```diff
@@ -155,6 +155,7 @@ sub doc_fullurl {
 # Parsers for other keys (basenames) will only run on the matching file.
 my %parser_map = (
     'index-faq'       => ['parse_faq'],
+    'index-language'  => ['parse_faq'],
```

Comment:
Yep, I guess that parser handles this content OK? I can do a screenshots run, we'll so how it all looks.